app/components/post/post.js

```diff
@@ -41,6 +41,7 @@ class Post extends PureComponent {
         intl: intlShape.isRequired,
         style: ViewPropTypes.style,
         post: PropTypes.object,
+        postId: PropTypes.string.isRequired, // Used by container // eslint-disable-line no-unused-prop-types
```

Comment:
if you're not going to use this then there's no need to declare it

Reply:
I think it's a good idea to do this so that callers know what they might need to pass as a prop. In this case, it was actually causing me problems not to have this since I didn't realize I wasn't passing this prop somewhere